### PR TITLE
Bugfix enable startup php extention

### PR DESCRIPTION
**Pull request by MaWich**
BugFix for Issue `#24`

### CODE REVIEW

File: docker/shared_steps/enable_php_modules.sh [hunk 1]
```diff
@@ -1,12 +1,13 @@
...
+[ ! -z "${PHP_EXTENSIONS_STARTUP_ENABLE}" ] || return 0
 
 for ext in $PHP_EXTENSIONS_STARTUP_ENABLE; do
     sectionText "Enable PHP extension $ext"
-    docker-php-ext-enable $ext
-    if [ -e "/usr/local/etc/php/mods-available/$ext.ini" ]; then
-        ln -fs /usr/local/etc/php/mods-available/$ext.ini /usr/local/etc/php/conf.d/20-$ext.ini
+    normalized_ext=`echo "$ext" | cut -d- -f1 | tr '[:upper:]' '[:lower:]'`
+    docker-php-ext-enable $normalized_ext
+    if [ -e "/usr/local/etc/php/mods-available/$normalized_ext.ini" ]; then
```
Comment (tonyfahrion) [inline docker/shared_steps/enable_php_modules.sh]: doesn't do `docker-php-ext-enable` exactly the same?

Comment (MaWich) [reply docker/shared_steps/enable_php_modules.sh]: Please indicate what exactly you mean ...

Comment (tonyfahrion) [reply docker/shared_steps/enable_php_modules.sh]: in line 9-11 you create a symlink in order to enable the php extension. I thought that this is already done by the command in line 8

File: .bumpversion.cfg [hunk 2]
```diff
@@ -1,5 +1,5 @@
 [bumpversion]
-current_version = 1.1.11
+current_version = 1.1.12
```
Comment (tonyfahrion) [inline .bumpversion.cfg]: hmm, did you use the tool `bump2version`? 🤔 
Because there are many more files, which need to be bumped

Comment (tonyfahrion) [reply .bumpversion.cfg]: this one: https://github.com/c4urself/bump2version

**Comment by MaWich** [comment]
i don't see `bump2version` in this source tree ... sorry can't execute this. :)

**Pull request merged**